app/src/marketplace/styles/pcss/buttons.pcss

```diff
@@ -156,14 +155,19 @@
     letter-spacing: 1px;
 
     &:--enter {
-      box-shadow: 0 0 10px rgba(0, 0, 0, 0.15);
       color: var(--greyDark);
+      background-color: rgba(255, 255, 255, 1);
```

Comment:
Isn't it just `#ffffff` or `white`?

Reply:
I think it helps show the relationship with `background-color: rgba(255, 255, 255, 0.9)`?

Reply:
Yep, exactly like that Tim. I think that you can better see the opacity change.